packages/api/client.ts

```diff
@@ -68,22 +68,49 @@ export const getOperators = async (dbClient: Kysely<Database>, input: OperatorQu
 export type StopsQueryInput = {
     atcoCodes?: string[];
     naptanCodes?: string[];
+    commonName?: string;
+    adminAreaCode?: string;
     page?: number;
 };
 
 export const getStops = async (dbClient: Kysely<Database>, input: StopsQueryInput) => {
     logger.info("Starting getStops...");
 
-    const STOPS_PAGE_SIZE = 50;
+    const STOPS_PAGE_SIZE = 1000;
```

Comment:
So we should be able to do something like `const STOPS_PAGE_SIZE = process.env.IS_LOCAL === "true" ? 50 : 1000;` and then in the stack we can add an env var: `IS_LOCAL: !["test", "preprod", "prod"].includes(stack.stage) ? "true" : "false",`